jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735NodeListXY.java

```diff
@@ -1,31 +1,28 @@
 package us.dot.its.jpo.ode.plugin.j2735;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-
 import us.dot.its.jpo.ode.plugin.asn1.Asn1Object;
 
 public class J2735NodeListXY extends Asn1Object {
 	/**
 	 * 
 	 */
 	private static final long serialVersionUID = 1L;
-	private J2735NodeSetXY nodes;
-	private J2735ComputedLane computed;
+	private J2735NodeXY[] nodes;
+	private J2735ComputedLane[] computed;
 
-	@JsonProperty("nodes")
-	public J2735NodeSetXY getNodes() {
+	public J2735NodeXY[] getNodes() {
 		return nodes;
 	}
 
-	public void setNodes(J2735NodeSetXY nodeList) {
-		this.nodes = nodeList;
+	public void setNodes(J2735NodeXY[] nodes) {
+		this.nodes = nodes;
 	}
 
-	public J2735ComputedLane getComputed() {
+	public J2735ComputedLane[] getComputed() {
```

Comment:
Just did a spot check here, didn't look at all the rest, but noticed that ComputedLane is not a SEQUENCE-OF (unlike NodeSetXY).  It shouldn't be an array.

Reply:
![image](https://github.com/user-attachments/assets/c97d7673-a8c4-4229-9ef5-4ed7b1ce5a9c)
Is it not though?

Reply:
It's a SEQUENCE which is more like a C Struct or Java POJO.  SEQUENCE "OF" is not the same and is like an array.  Confusing terminology.

Reply:
Spoke with Ivan offline, the SEQUENCE is a ASN1 XML keyword for an object while SEQUENCE OF is the array. This will be addressed.